linux_os/guide/system/network/network-nftables/nftables_rules_permanent/tests/invalid_top_level_config_file.fail.sh

```diff
@@ -0,0 +1,4 @@
+# variables = var_nftable_master_config_file=/etc/sysconfig/nftables.conf
+
+# make sure that variable is set correctly on new platform adding
+echo 'include /some/invalid/path' > "${var_nftable_master_config_file}"
```

Comment:
This surely won't work. The variable `${var_nftable_master_config_file}` is undefined. You can't use the XCCDF Value variables as variables in test scenarios, they're completely unrelated. By setting `# variables = var_nftable_master_config_file=/etc/sysconfig/nftables.conf` in the header you define that the XCCDF Value `var_nftable_master_config_file` is set to `/etc/sysconfig/nftables.conf` . That gives you an assumption for your test scenario but it doesn't define any bash variables. So you need to set that explicitly.
```suggestion
echo 'include /some/invalid/path' > "/etc/sysconfig/nftables.conf"
```
There is a similar problem in also other test scenarios.